proto/c1/config/rules.proto

```
@@ -0,0 +1,194 @@
+syntax = "proto3";
+
+package c1.config.v1;
+option go_package = "github.com/conductorone/baton-sdk/pb/c1/config/v1";
+
+import "google/protobuf/descriptor.proto";
+import "google/protobuf/duration.proto";
```

Buf (1.47.2)

Import "google/protobuf/duration.proto" is unused.

(IMPORT_USED)